src/commands/with_cache.yml

```diff
@@ -40,7 +44,11 @@ steps:
         if [ -n "<< parameters.settings_file >>" ]; then
           set -- "$@" --settings "<< parameters.settings_file >>"
         fi
-        << parameters.maven_command >> org.apache.maven.plugins:maven-dependency-plugin:3.1.2:go-offline "$@"
+  - run:
+      name: Verify dependencies
+      working_directory: << parameters.app_src_directory >>
+      when: << parameters.verify_dependencies >>
```

Comment:
Improper use of when: https://circleci.com/docs/2.0/configuration-reference/#the-when-attribute

You may instead be looking for this: https://circleci.com/docs/2.0/configuration-reference/#the-when-step-requires-version-21

Reply:
@KyleTryon I've updated to use the latter type, let me know if that looks right.